sdk/search/Build-SearchPackages.ps1

```diff
@@ -1,7 +1,7 @@
 $scriptDir = Split-Path $MyInvocation.MyCommand.Path -Parent
 $engDir = "$scriptDir\..\..\eng\"
 
-msbuild "$scriptDir\..\..\build.proj" /t:CreateNugetPackage /p:Scope="search"
+dotnet msbuild "$scriptDir\..\..\mgmt.proj" /t:CreateNugetPackage /p:Scope=Search
```

Comment:
FYI - My goal is to have get the mgmt and data plane builds working similar so all you should need to do is to call "dotnet test" and "dotnet pack" on the root build.proj with the scope set to search. Currently not all the work is done to enable that but it is my eventual goal.

Reply:
That would be awesome! In the meantime, if we haven't achieved this already, please ensure that all the Search libraries are building in the CI build so that build breaks are caught early (ideally before they hit master and disrupt my team).